src/evse_man.cpp

```diff
@@ -445,6 +439,12 @@ unsigned long EvseManager::loop(MicroTasks::WakeReason reason)
   {
     _evaluateTargetState = false;
     setTargetState(_targetProperties);
+
```

Comment:
I am not super keen on this, will leave it for now but we need to consider how to make this more generic, I would prefer not to have these special cases in the EVSE manager... We probably need to give the clients text names and event them  all or something like that...